### DISCUSSION TITLE
[project] 5/3(목) todolist

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#122] [General]
오늘은 todolist를 끝내 볼려고 합니다. 보통 todolist 초반에는 userDefault를 사용하지만 오늘 저는 CoreData로 하는 법을 알려드릴까합니다. 물론 저 1학년 때 우리두라는 프젝이 todolist 였지만 저의 코드가 보여주기에는 너무 창피하기에 아래 주소를 통해 들어가면 CoreData를 사용한 주소가 있습니다. 일단 코드를 받아서 보시고 아래는 제가 참고 했던 블로그들 입니다.

아래 url은 저의 CoreData 코드들입니다.
[박준하 CoreData 코드](https://github.com/SecureMarshmallow/SecureMarshmallow_iOS_V3/tree/master/SecureMarshmallow_V3/Source/ViewControllers/Lock)

[참고 주소](https://silver-g-0114.tistory.com/71)
[참고 주소](https://hongssup.tistory.com/146)

모르는 것 있으면 자주자주 물어봐주세요 저도 도움이 많이 됩니다!!
제가 압박을 주는 것은 아니지만 가능한이면 오늘 까지 끝냈으면 좋겠습니다. 커밋은 가능한이면 자세히 자주 해주세요. 😁(제가 보고있습니다!!!)

**Comment by Eunho0922** [comment]
넵 감사합니다
